src/css/components/count_status.scss

```diff
@@ -5,7 +5,7 @@
 
 .count_status {
   display: inline-block;
-  height: 0.6rem;
+  height: 1.1rem;
```

Comment:
What's with the non-round number? Could this be based on our grid-sizing hierarchy?